src/components/Navigation/Navigation.scss

```diff
@@ -0,0 +1,32 @@
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 95%;
+
+  .home-logo {
+    font-family: "Arial", sans-serif;
+    font-size: 1.5rem;
+    font-weight: bold;
+    color: black;
+    text-transform: uppercase;
+    text-decoration: none;
+    letter-spacing: 0.1rem;
+  }
+
+  .auth-links-container {
+    display: flex;
+    gap: 0.5rem;
+
+    .auth-link {
+      background-color: black;
+      color: white;
```

Comment:
some of you are using hex colors, others - named colors, so better to create a color scheme you want to use across the whole app and reuse the color variables